### PR TITLE
Converted calc_liq_relaxation_timescale to C++/Kokkos

### PR DESCRIPTION
**Pull request by jeff-cohere**
This follows #290 , and includes converted code and tests for `calc_liq_relaxation_timescale`.

It also adds `revap_table` to the set of tables initialized for Kokkos from Fortran.

### CODE REVIEW

**Comment by jgfouca** [comment]
Please rebase this branch off of latest master and resolve conflicts.

**Comment by jeff-cohere** [comment]
The failures on Bowman and White are "no space left on device" errors. I'm not sure what to make of the one on Blake: `Build type debug_nopack_fpe failed`. Could that be a divide by zero thing or another FPE-related error?

File: components/scream/src/physics/p3/p3_functions_calc_liq_relaxation_timescale_impl.hpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                                const Spack& cdist, const Spack& qr_incld, const Spack& qc_incld,
+                                Spack& epsr, Spack& epsc)
+{
+  constexpr Scalar qsmall = C::QSMALL;
+  constexpr Scalar pi = C::Pi;
+
+  const auto qr_not_small = (qr_incld >= qsmall);
+  Table3 table;
+  lookup(qr_not_small, mu_r, lamr, table);
+  epsr.set(qr_not_small, sp(2.0) * pi * cdistr * rho * dv *
```
Comment (jgfouca) [inline components/scream/src/physics/p3/p3_functions_calc_liq_relaxation_timescale_impl.hpp]: For integers, use integer literals so you don't need the sp macro.

File: components/scream/src/physics/p3/p3_functions_calc_liq_relaxation_timescale_impl.hpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+           (f1r * tgamma(mu_r + sp(2.0))/lamr +
+            f2r * sqrt(rho/mu) * cbrt(sc) *
+            apply_table(qr_not_small, revap_table, table)));
+  const auto qr_small = not qr_not_small;
+  epsr.set(qr_small, 0);
+
+  const auto qc_not_small = (qc_incld >= qsmall);
+  epsc.set(qc_not_small, sp(2.0) * pi * rho * dv * cdist);
```
Comment (jgfouca) [inline components/scream/src/physics/p3/p3_functions_calc_liq_relaxation_timescale_impl.hpp]: Same

File: components/scream/src/physics/p3/p3_functions_calc_liq_relaxation_timescale_impl.hpp (hunk 1 above)
Comment (PeterCaldwell) [reply components/scream/src/physics/p3/p3_functions_calc_liq_relaxation_timescale_impl.hpp]: I like the integer literal trick!

**Comment by jeff-cohere** [comment]
Hey @jgfouca , do you want me to handle the merge conflicts, or do you want to finish this one up? Thanks for your help on this!

**Comment by jgfouca** [comment]
I got it.

**Pull request merged**